masonry/src/widgets/button.rs

```diff
@@ -302,6 +303,16 @@ impl Widget for Button {
     }
 }
 
+impl HasProperty<DisabledBackground> for Button {}
+impl HasProperty<ActiveBackground> for Button {}
+impl HasProperty<Background> for Button {}
+impl HasProperty<HoveredBorderColor> for Button {}
+impl HasProperty<BorderColor> for Button {}
+impl HasProperty<BorderWidth> for Button {}
+impl HasProperty<CornerRadius> for Button {}
+impl HasProperty<Padding> for Button {}
+impl HasProperty<BoxShadow> for Button {}
```

Comment:
I've put these below everything, they could reasonably be also directly below the widget structs. I think that would make sense if we're dedicating ourselves more towards a Struct of Arrays (Components of ECS) architecture.